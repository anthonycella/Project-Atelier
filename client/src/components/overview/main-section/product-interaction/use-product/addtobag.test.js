import renderer from 'react-test-renderer';
import AddToBag from './addtobag.jsx';

it('Add To Bag renders to the page', () => {
  const component = renderer.create(
    <AddToBag />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
