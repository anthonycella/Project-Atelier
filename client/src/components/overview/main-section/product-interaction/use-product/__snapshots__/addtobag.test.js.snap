// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add To Bag renders to the page 1`] = `
<div
  className="sc-bczRLJ bvsNxk"
  onClick={[Function]}
>
  ADD TO CART
</div>
`;
